notebooks/05_operationalize/create_and_configure_cluster.ipynb

```diff
@@ -0,0 +1,718 @@
+{
+ "cells": [
+  {
```

Comment:
question: is this notebook a general way to configure databricks with Recommenders or it this only for the reference architecture?

Reply:
started as being specific for the reference architecture, but with the deprecation of the runtime, i tried to make it more general. There could be two different versions. One for general purpose use of Recommenders, one for the more extensive end-to-end, because the end-to-end does have additional lib dependencies (azure-ml, pydocumentdb, the Cosmos jar)

Reply:
removed notebook for now based on CLI approach